### PR TITLE
Add info needed to add Furious to the public pypi

### PR DESCRIPTION
**Pull request by beaulyddon-wf**
Since Furious is open source we should be good people and add our lib to pypi so
others can easily use it via pip/easy_install and not be required to build from
source.

I've added the required info to setup.py and setup.cfg. Wasn't much. Also added
the generated pypi files to the .gitignore so we don't push those up.

@robertkluin @nickjoyce-wf @shawnrusaw-wf @tylertreat @tannermiller-wf 

FYI @chadknight-wf for the request

### CODE REVIEW

**Comment by rmconsole-wf** [comment]
## SOC 1 Checklist
- [ ] All commits reviewed (ac1875cd9bed29416adc2616a79a970c884c02ff)
- [ ] QA Review
- [x] Security Review Completed if Applicable:  (N/A)

To remain in SOC1 Compliance the following users cannot merge this PR:
* beaulyddon-wf

***
*Last updated on Friday, May 01 12:27 PM CST*

**Comment by robertkluin** [comment]
+1

**Comment by tannermiller-wf** [comment]
+1

**Comment by tylertreat-wf** [comment]
+1

File: setup.py [hunk 1]
```diff
@@ -21,16 +21,17 @@ def get_version():
     author_email='robert.kluin@webfilings.com',
```
Comment (chadknight-wf) [inline setup.py]: might be a good time to update these to workiva.com and Workiva

**Comment by beaulyddon-wf** [comment]
Updated info. Can I get follow up reviews ...
@robertkluin @nickjoyce-wf @shawnrusaw-wf @tylertreat @tannermiller-wf

**Comment by shawnrusaw-wf** [comment]
+1

**Comment by nickjoyce-wf** [comment]
+1

**Comment by tylertreat-wf** [comment]
+1

**Comment by tannermiller-wf** [comment]
+1

**Pull request merged**